exporting/mongodb/README.md

```diff
@@ -0,0 +1,28 @@
+# MongoDB exporting connector
+
```

Comment:
Let's add a sentence at the beginning that explains _why_ someone might care about this.

```
You can use the MongoDB connector and the experimental [exporting engine](../README.md) to archive your agent's metrics to a MongoDB database for long-term storage, further analysis, or correlation with data from other sources.
```

Reply:
Added